src/main/java/uk/gov/pay/webhooks/message/WebhookMessage.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.pay.webhooks.message;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import uk.gov.pay.webhooks.ledger.model.LedgerTransaction;
+import uk.gov.pay.webhooks.message.dao.entity.WebhookMessageEntity;
+import uk.gov.pay.webhooks.queue.EventMessage;
+import uk.gov.pay.webhooks.queue.InternalEvent;
+import uk.gov.pay.webhooks.util.MicrosecondPrecisionInstantDeserializer;
+import uk.gov.pay.webhooks.webhook.dao.entity.WebhookEntity;
+import uk.gov.service.payments.commons.api.json.ApiResponseInstantSerializer;
+import uk.gov.service.payments.commons.api.json.MicrosecondPrecisionDateTimeDeserializer;
+
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+
+public record WebhookMessage(String id,
+                             @JsonProperty("created_date") @JsonSerialize(using = ApiResponseInstantSerializer.class) Instant createdDate,
```

Comment:
I think unless this is being automatically created with an ObjectMapper (seem to recall there was some corner case there) all of these explicit property annotations can be replaced by annotating the record with 
```java
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```

Reply:
Good shout, have done this in 5ee32fa.